lib/Doctrine/MongoDB/EagerCursor.php

```diff
@@ -50,6 +50,13 @@ class EagerCursor implements CursorInterface
     protected $initialized = false;
 
     /**
+     * Whether the cursor started iterating. This is necessary to get getNext() and hasNext() to work properly
```

Comment:
I would put "This is needed to implement getNext() and hasNext()" as its own paragraph (after two line breaks).